semver.md

```diff
@@ -121,7 +121,7 @@ have the same precedence. Examples: 1.0.0-alpha+001, 1.0.0+20130313144700,
 
       Example: 1.0.0 < 2.0.0 < 2.1.0 < 2.1.1.
 
-   1. When major, minor, and patch are equal, a pre-release version has lower
+   1. When major, minor, and patch are equal, a pre-release version has higher
       precedence than a normal version:
 
       Example: 1.0.0-alpha < 1.0.0.
```

Comment:
this reads like an incorrect change to me - 1.0.0 is higher than 1.0.0-alpha, as the previous text states, which means a pre-release version indeed has a *lower* precedence.

Reply:
1.0.0 is higher in ordering, and therefore has a lower *precedence* (the quality of preceding) than 1.0.0-alpha. It would be more clear to only use the word "precedence" to refer to the elements of the version string (e.g. "When comparing, the pre-release identifier has a lower precedence than the major/minor/patch numbers."), as opposed to using it to refer to the entire version string. But, as I mentioned, that would be a larger change.